File: week1/mario_less/mario.c
#include <stdio.h>
#include <cs50.h>

int	main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    } while (n < 1 || n > 8);
//  printf ("Stored: %i\n", n);
    for (int i = 1; i <= n; i++)
    {
        int j = n - i;
        while (j > 0)
        {
            printf(" ");
            j--;
        }
        j = i;
        while (j > 0)
        {
            printf("#");
            j--;
        }
        printf ("\n");
    }
    return (0);
}

File: week6/mario/less/mario.py
from cs50 import get_int

c = get_int('Height: ')
while (c < 1 or c > 8):        # Loop until wrong conditions
    c = get_int('Height: ')

for i in range(1, c+1):
    j = c-i
    while(j > 0):
        print(' ', end='')
        j -= 1
    j = i
    while(j > 0):
        print('#', end='')
        j -= 1
    i += 1
    print('\n', end='')
